src/roiextractors/extractors/inscopixextractors/inscopixsegmentationextractor.py

```diff
@@ -24,39 +21,145 @@ def __init__(self, file_path: PathType):
 
         Parameters
         ----------
-        file_path: str
+        file_path: str or PathType
```

Comment:
PathType is already Union[str, Path]
```suggestion
        file_path: str or Path
```